services/booking-api/lambdas/cancel.js

```diff
@@ -0,0 +1,18 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  const bookingId = event.pathParameters.id;
+  const body = { bookingId };
```

Comment:
This looks somewhat confusing to me.
I would interpret booking Id to contain the actual id for the booking?

Curious what the 'body' means?

Maybe the pathParameter - id is lying in some regard?

Reply:
My bad. Ignore